drv/bl_radio/bl_radio_default.c

```diff
@@ -32,34 +32,36 @@
 #define RADIO_INTERRUPT_PRIORITY 1
 #endif
 
-#define RADIO_TIFS          0U  ///< Inter frame spacing in us. zero means IFS is enforced by software, not the hardware
+#define RADIO_TIFS 0U ///< Inter frame spacing in us. zero means IFS is enforced by software, not the hardware
 
-#define RADIO_SHORTS_COMMON (RADIO_SHORTS_END_DISABLE_Enabled << RADIO_SHORTS_END_DISABLE_Pos) | \
-                            (RADIO_SHORTS_ADDRESS_RSSISTART_Enabled << RADIO_SHORTS_ADDRESS_RSSISTART_Pos) | \
-                            (RADIO_SHORTS_DISABLED_RSSISTOP_Enabled << RADIO_SHORTS_DISABLED_RSSISTOP_Pos)
+#define RADIO_SHORTS_COMMON (RADIO_SHORTS_END_DISABLE_Enabled << RADIO_SHORTS_END_DISABLE_Pos) |                 \
+                                (RADIO_SHORTS_ADDRESS_RSSISTART_Enabled << RADIO_SHORTS_ADDRESS_RSSISTART_Pos) | \
+                                (RADIO_SHORTS_DISABLED_RSSISTOP_Enabled << RADIO_SHORTS_DISABLED_RSSISTOP_Pos)
 
-#define RADIO_INTERRUPTS    (RADIO_INTENSET_ADDRESS_Enabled << RADIO_INTENSET_ADDRESS_Pos) | \
-                            (RADIO_INTENSET_END_Enabled << RADIO_INTENSET_END_Pos) | \
-                            (RADIO_INTENSET_DISABLED_Enabled << RADIO_INTENSET_DISABLED_Pos)
+#define RADIO_INTERRUPTS (RADIO_INTENSET_ADDRESS_Enabled << RADIO_INTENSET_ADDRESS_Pos) | \
+                             (RADIO_INTENSET_END_Enabled << RADIO_INTENSET_END_Pos) |     \
+                             (RADIO_INTENSET_DISABLED_Enabled << RADIO_INTENSET_DISABLED_Pos)
 
 #define RADIO_STATE_IDLE 0x00
-#define RADIO_STATE_RX   0x01
-#define RADIO_STATE_TX   0x02
+#define RADIO_STATE_RX 0x01
+#define RADIO_STATE_TX 0x02
 #define RADIO_STATE_BUSY 0x04
 
-typedef struct __attribute__((packed)) {
-    uint8_t header;                              ///< PDU header (depends on the type of PDU - advertising physical channel or Data physical channel)
-    uint8_t length;                              ///< Length of the payload + MIC (if any)
-    uint8_t payload[BL_BLE_PAYLOAD_MAX_LENGTH];  ///< Payload + MIC (if any) (BL_BLE_PAYLOAD_MAX_LENGTH > BL_IEEE802154_PAYLOAD_MAX_LENGTH)
+typedef struct __attribute__((packed))
+{
+    uint8_t header;                             ///< PDU header (depends on the type of PDU - advertising physical channel or Data physical channel)
+    uint8_t length;                             ///< Length of the payload + MIC (if any)
+    uint8_t payload[BL_BLE_PAYLOAD_MAX_LENGTH]; ///< Payload + MIC (if any) (BL_BLE_PAYLOAD_MAX_LENGTH > BL_IEEE802154_PAYLOAD_MAX_LENGTH)
```

Comment:
Any reason for changing this indentation? (makes a bit hard to review since there was no real change here, but the diff show it changed)